### ISSUE TITLE
Add root for intellisense to workspace settings

### ISSUE THREAD

**Issue by alexkrolick** [Microsoft/vscode#11915]
It would be nice to be able to configure Intellisense to assume the workspace root is something other than the current directory, so that absolute (`/...`) or root-less (`...`) requires would load their Intellisense definitions:

Example:

**Require path**
```
var Modal = require('shared/elements/Modal');
```

**Actual path relative to project root**
```
app/assets/javascripts/shared/elements/Modal.js
```

**Comment by dbaeumer** [comment]
That is available in TS 2.0

**Comment by alexkrolick** [comment]
Adding `"baseUrl": "app/assets/javascripts/"` to jsconfig.json seems to work on the Insiders build at least.

**Comment by waderyan** [comment]
Closing as this is solved in TS 2.0

**Comment by nishp1** [comment]
I am not seeing `baseUrl` work in either 1.7.2 or insider build.  Has the config changed?
